src/logi_transform.erl

```diff
@@ -136,7 +136,7 @@ logi_location_expr(Loc = #location{line_or_anno = LineOrAnno}) ->
        {atom, LineOrAnno, Loc#location.application},
        {atom, LineOrAnno, Loc#location.module},
        {atom, LineOrAnno, Loc#location.function},
-       {integer, LineOrAnno, LineOrAnno}
+       {integer, LineOrAnno, logi_transform_utils:line_or_anno_to_line(LineOrAnno)}
```

Comment:
When anno contains column, this made malformed abstract format values like `{integer, {10, 12}, {10, 12}}` wherer the third element must be an integer.